src/profile/new.rs

```diff
@@ -0,0 +1,100 @@
+use std::fmt::Debug;
+
+use clap::Parser;
+// use rand::Rng;
+// use sha2::{Digest, Sha256};
+// use stellar_strkey::StrkeyPublicKeyEd25519;
+
+use crate::{profile::store, HEADING_RPC, HEADING_SANDBOX};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    ProfileStoreError(#[from] store::Error),
+    #[error("profile already exists: {name}")]
+    ProfileAlreadyExists { name: String },
+}
+
+#[derive(Parser, Debug)]
+pub struct Cmd {
+    /// Name of the profile, e.g. "sandbox"
+    #[clap(long)]
+    name: String,
+
+    /// Overwrite any existing profile with the same name.
+    #[clap(long, short = 'f')]
+    force: bool,
+
+    /// File to persist ledger state (if using the sandbox)
+    #[clap(
+        long,
+        parse(from_os_str),
+        default_value = ".soroban/ledger.json",
+        conflicts_with = "rpc-url",
+        env = "SOROBAN_LEDGER_FILE",
+        help_heading = HEADING_SANDBOX,
+    )]
+    ledger_file: std::path::PathBuf,
+
+    /// RPC server endpoint
+    #[clap(
+        long,
+        requires = "network-passphrase",
```

Comment:
Being new to the ecosystem, what it he point of this passphrase if the endpoints are unique?

Also this feels error prone since it depends on a long string.

Reply:
Everything for a network includes the network passphrase in the signature payload. It makes things signed for Testnet invalid for Pubnet, and vice versa. It also allows folks to run their own networks, and even if you use the same keys across all, the things you sign will only be valid for the network they are intended.

This is similar to a "chain ID" in some other networks.